<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuchergebnisDialog</class>
 <widget class="QDialog" name="SuchergebnisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="searchLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name des Passagiers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassagierName"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPlayer">
       <property name="title">
        <string>Reisepläne als Player</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutPlayer">
        <item>
         <widget class="QTableWidget" name="tableWidgetPlayer">
          <column>
           <property name="text">
            <string>PlanNr.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PlanName</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ReiseBeginn</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ReiseEnde</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPassagier">
       <property name="title">
        <string>Reisepläne als Passagier</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutPassagier">
        <item>
         <widget class="QTableWidget" name="tableWidgetPassagier">
          <column>
           <property name="text">
            <string>PlanNr.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PlanName</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ReiseBeginn</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ReiseEnde</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>